akka-stream-tests/src/test/scala/akka/stream/scaladsl/StreamRefsSpec.scala

```diff
@@ -353,7 +380,7 @@ class StreamRefsSpec(config: Config) extends AkkaSpec(config) with ImplicitSende
       remoteActor ! "receive-subscribe-timeout"
```

Comment:
row above ^ says "now", should it be "not"?

Reply:
thanks